### PR TITLE
New Timeline section for Kaui Guide

### PR DESCRIPTION
**Pull request by ghost**
A short section. Just 2 pages in the PDF version. LMK if you think we need more.

### CODE REVIEW

**Review by sbrossie — APPROVED** [review]
I browsed through it, it looks great. Since there is a lot of subtle stuff around pointing to images, and updating things like `width`, .. it's hard to really picture what will be rendered.

Also, looks like you have a conflict to resolve but I know you are now a got power user , so should be good ;-)

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 1]
```diff
@@ -1254,25 +1449,30 @@ image::ProcessChargeback.png[width=450,align="center"]
...
+
+To see the Timeline, open an account and click *Timeline* on the sub-menu:
+
+image::TimelinePage.png[width=850,align="center"]
 
-// How to view/interpret the timeline. What it shows.
+On the Timeline page, you can:
 
-// The Timeline also includes information about payment retries (When an account is unpaid, Kill Bill (the payment retry system) retries payments per the scheduled configuration.
+* _Filter the events_: Make selections in the *Filter by event* or *Filter by bundle* fields to display only specific events (e.g., `PURCHASE`, `INVOICE`) and/or by a specific subscription bundle.
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1499]: Minor correction: `Filter by event` should be `Filter by event type`

**Review by reshmabidikar — COMMENTED** [review]
The PDF does not open, so I directly reviewed the `Kaui_Guide_Draft.adoc` file and looks good to me (I was not able to review images as they are not visible in the `adoc` file but I think this should be fine since it is a small section). 

Another suggestion: Have you considered splitting up the `adoc` file into separate smaller documents corresponding to each section? For example, take a look at the [userguide_subscription.adoc](https://github.com/killbill/killbill-docs/blob/v3/userguide/subscription/userguide_subscription.adoc). Each section is present in a separate file within the [includes ](https://github.com/killbill/killbill-docs/tree/v3/userguide/subscription/includes)folder. The `userguide_subscription.adoc` just references each section via the [include](https://github.com/killbill/killbill-docs/blob/v3/userguide/subscription/userguide_subscription.adoc#overview) directive. I recently followed this approach for a document that I was working on and found that it makes the document easier to edit/maintain.

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 1 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1499]: This has been fixed

**Comment by ghost** [comment]
I agree the guide is getting unwieldy. I'll take some time to break it into separate files.

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**